css/base.css

```diff
@@ -1,10 +1,19 @@
 *{
   width: 100%;
```

Comment:
width and heigth 100% should not be under the * selector, this will cause unexpected behavior.